doc/release-notes/release-1.4.1.md

```diff
@@ -0,0 +1,23 @@
+# khmer v1.4.1 release notes
+
+This is the v1.4.1 release of khmer. Due to the Python 3 migration we need to
```

Comment:
suggest referring to issue # here (for Python 3 migration) OR changing it to be "Due to the upcoming Python 3 compatibility that will be part of khmer 2.0" or some more info.